plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -227,6 +227,7 @@ export default Component.extend({
     }
   },
 
+  @debounce(100)
```

Comment:
prevents two very fast fetching when having a targetMessageId: from lastReadId and then from targetMessageId